lib/betamocks/response_cache.rb

```diff
@@ -14,8 +15,10 @@ def initialize(env:, config: nil)
 
     def load_response
       raise IOError, "Betamocks cache_dir: [#{Betamocks.configuration.cache_dir}], does not exist" unless File.directory?(Betamocks.configuration.cache_dir)
-      if File.exist?(file_path(@generated_file_name))
-        Faraday::Response.new(load_env(@generated_file_name))
+      if File.exist?(optional_locator_file_path(@generated_file_name))
+        Faraday::Response.new(load_env(optional_locator_file_path(@generated_file_name)))
+      elsif File.exist?(file_path(@generated_file_name))
+        Faraday::Response.new(load_env(file_path(@generated_file_name)))
       else
         Betamocks.logger.warn "Mock response not found: [#{file_path(@generated_file_name)}]"
```

Comment:
I know you didn't write this but shouldn't we just throw an Exception if the file doesn't exist?

Reply:
Possibly, I have a feeling that would break a bunch of stuff. Right now it serves a 'default' response that looks like a user, so there are probably a number of non-mocked test users that rely on this. (Not that this is a good solution, but just one I won't be able to solve in this PR, ha